charts/backstage/Chart.yaml

```
@@ -50,4 +50,4 @@ sources: []
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 # Note that when this chart is published to https://github.com/openshift-helm-charts/charts
 # it will follow the RHDH versioning 1.y.z
-version: 4.2.7
+version: 4.2.8
```

```suggestion
version: 4.2.7
```

No need to bump the Chart version then.